database/seeders/LaratrustSeeder.php

```diff
@@ -483,5 +487,13 @@ public function run() {
             'permission_id' => 10,
             'role_id' => 13
         ]);
+        DB::table('permission_role')->insert([
+            'permission_id' => 2,
+            'role_id' => 15
+        ]);
```

Comment:
Just caught this after QA - does this give more permissions to the marketing role than we want? If we only want the 3 listed in the story, it may be better to add them to the email group and manually change the code so they can see announcements and the calendar

Reply:
I considered this, but yeah you are probably right here. Even if it isn't a problem now, I want to future-proof this.